overrides/macos-override.json

```diff
@@ -522,7 +522,13 @@
             "state": "enabled"
         },
         "ampLinks": {
-            "state": "enabled"
+            "state": "enabled",
+            "exceptions": [
+                {
+                    "domain": "publicmediasignin.org",
+                    "reason": "https://github.com/duckduckgo/privacy-configuration/issues/2312"
```

Comment:
This needs to point to the PR, which should be /pull/2312 rather than /issues, I think.

Reply:
Sorry changes just made